products/sle/main.pm

```diff
@@ -767,6 +767,9 @@ elsif (get_var("VIRT_AUTOTEST")) {
     }
     if (get_var("VIRT_PRJ1_GUEST_INSTALL")) {
         loadtest "virt_autotest/guest_installation_run";
+        loadtest "virt_autotest/set_config_as_glue";
```

Comment:
Can you help me understand what "set configuration as glue" means, please?

Reply:
There is the background, we need to reuse the QAM code - xen.pm which is the testing entry both our virt_autotest and QAM  virtualization testing, had defined QAM required guests list already. 
Now, we need to define our virt_autotest required guest from xen.pm, so there was created set_config_as_gule to call xen.pm to set our required configuration as gule to reuse QAM code here. 
Hope that help you understand what "set configuration as glue" means. 
Next, we will work with QAM team together to add new virtualization test run, refer to 
 https://etherpad.suse.de/p/virtualization-testing-coverage, so we'd better to add new required configuration and setting to set_config_as_gule file.